app/views/moirai/translation_files/_form.html.erb

```diff
@@ -1,9 +1,9 @@
 <span 
   contenteditable 
   data-action="blur->moirai-translation#submit click->moirai-translation#click"
-  style="border: 1px dashed #1d9f74; min-width: 30px; display: inline-block;" 
+  style="border: 1px dashed #1d9f74; min-width: 30px; display: inline-block; <%= 'color: red;' if missing_translationf %>" 
   data-moirai-translation-key-value="<%= key %>"
   data-moirai-translation-locale-value="<%= locale %>"
   data-controller="moirai-translation">
-  <%= value %>
+  <%= value.to_s.gsub("<", "&lt;").gsub(">", "&gt;").html_safe %>
```

Comment:
Neither 
`<%= html_escape(value %>`
nor
`<%= h(value %>`
seem to do the trick.